kfcluster.go

```diff
@@ -5,18 +5,20 @@ import (
 	"encoding/json"
 	"fmt"
 	"strings"
+	"time"
 )
 
 // KfCluster represents a cluster with Kubeflow installed.
 type KfCluster struct {
-	ID            string `json:"id"`
-	Name          string `json:"name,validate:required"`
-	NetworkID     string `json:"network_id,validate:required"`
-	FirewallID    string `json:"firewall_id"`
-	Size          string `json:"size"`
-	KubeflowReady string `json:"kubeflow_ready"`
-	DashboardURL  string `json:"dashboard_url"`
-	Namespace     string `json:"-"`
+	ID            string    `json:"id"`
+	Name          string    `json:"name"`
+	NetworkID     string    `json:"network_id"`
+	FirewallID    string    `json:"firewall_id"`
+	Size          string    `json:"size"`
+	KubeflowReady string    `json:"kubeflow_ready"`
+	DashboardURL  string    `json:"dashboard_url"`
+	CreatedAt     time.Time `json:"created_at"`
+	Namespace     string    `json:"-"`
```

Comment:
Namespace?

Reply:
I think is a refuse. I'll remove it. It caught my attention too